es/ponyfill.js

```diff
@@ -0,0 +1,17 @@
+export default function symbolObservablePonyfill(root) {
+	let result;
```

Comment:
I don't precisely follow. So this has been reimplemented with ES2015 compliant code, however what I *think* you're saying is that this should be ES5 with ES2015 module `import` and `export` clauses? Is that correct? If so, why?